ras-collection-instrument/settings.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+''' This file is the main configuration for the Collection Instrument Service.
+    It contains a full default configuration
+    All configuration may be overridden by setting the appropriate environment variable name. '''
+
+DEBUG = False
+TESTING = False
+CSRF_ENABLED = True
+SECRET_KEY = 'this-really-needs-to-be-changed'
+dbname = "ras_collection_instrument"
+SQLALCHEMY_DATABASE_URI = os.environ.get('SQLALCHEMY_DATABASE_URI', "postgresql://" + dbname + ":password@localhost:5431/postgres")
+
+
+#OAuthConfig
```

Comment:
The original didn't have everything commented out as this one does. I assumed it was therefore redundant and should be removed